### PR TITLE
Bobby  US3-Event tab(AC #2)

### PR DESCRIPTION
**Pull request by ik007g**
1. Updated TestBase (added EventTabPage class)
2. Added linkTest (US3)
3. Created EventTabPage class
4. added new attachLink method to EventTabPage

### CODE REVIEW

File: src/test/java/com/nextbasecrm/tests/EventTabTests.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    @BeforeMethod
+    public void setUpMethod(){
+        loginPage.login(ConfigurationReader.getProperty("help_desk_username"), ConfigurationReader.getProperty("help_desk_password"));
+    }
+    @Test (description = "User should be able to attach link by clicking on the link icon",
+    dataProvider = "links")
+    public void linkTest(String linkText, String linkUrl) {
+      eventTabPage.attachLink(linkText, linkUrl);
+      wait.until(ExpectedConditions.textToBePresentInElement(eventTabPage.savedLinkWithText, linkText));
+      Assert.assertTrue(eventTabPage.savedLinkWithText.getAttribute("innerHTML").equals(linkText));
```
Comment (mikat2122) [inline src/test/java/com/nextbasecrm/tests/EventTabTests.java:21]: Great

File: src/test/java/com/nextbasecrm/pages/EventTabPage.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    public WebElement saveBtn;
+
+    //LINK IS LOCATED WITHIN THE FRAME, SO YOU NEED TO LOCATE FRAME FIRST
+    @FindBy(xpath = "(//iframe[@class='bx-editor-iframe'])[2]")
+    public WebElement iframeWithLink;
+
+    //MY LINK
+    @FindBy(xpath ="//body/a" )
+    public WebElement savedLinkWithText;
+
```
Comment (mikat2122) [inline src/test/java/com/nextbasecrm/pages/EventTabPage.java:28]: You need to add method description

**Review by mikat2122 — COMMENTED** [review]
Please update according to comments

**Review by mikat2122 — APPROVED** [review]

**Review by pshilin — APPROVED** [review]
Great job!

**Pull request merged**